CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+# Contributing
+
+## Are you writing a migration to change Publishing API data?
+
+We have a lot of migrations added to the Publishing API to change data, our
+long term plan is to reduce these so that these can be done via the API itself
+and not need migrations. Until we are at that point please use the following
+guidelines when creating migrations.
+
+### Run the migration on your machine first
+
+At the very least this will ensure that the code runs and that your data looks
+correct, but it also allows you to get an idea for how long the migration will
+take to finish. You should then include that information in the PR to help with
+whoever is doing the deployment.
+
+We also encourage you to run and check the results of the migration on
+integration, since the data there will be closer to production.
+
+### Always include the `schema.rb` file
+
+Even if the change is just the timestamp, it's important to include this file
+in your commit otherwise when trying to use the app in testing an error saying
+there are pending migrations will appear.
+
+### Target only content related to the application you are affecting
+
+Be as specific as possible when updating content, for example, include a
+condition on the `publishing_app` even if it seems unlikely there would be
+content in any other state.
```

Comment:
I'd be a bit firmer here, more that mistakes can and do happen the least you can do is scope your query just to the publishing_app.